### PR TITLE
Fix/black jack clutter

### LINKED ISSUE #15: Print Statements are cluttered and inconsitent (has to look good and also improve the code readability.)
In the Python BlackJack project Print Statements are cluttered and inconsistent
image

- [ ] 1. It has to look good
- [ ] 2. Improve the code readability.

### PR DESCRIPTION
**Pull request by bakanabaka**
##What does this PR do
With this output is as desired

Fixes https://github.com/yashkite/Code-Practice/issues/15

### CODE REVIEW

File: Small Projects/Python/Black Jack/run.py [hunk 1]
```diff
@@ -1,186 +1,233 @@
...
+        5,
+        6,
+        7,
+        8,
+        9,
+        10,
+        10,
+        10,
+        10,
+    ]
```
Comment (yashkite) [inline Small Projects/Python/Black Jack/run.py:100]: I like the old way they are presented, its more readable.
image

**Review by yashkite — CHANGES_REQUESTED** [review]
thanks for the PR, changes requested !!!

**Comment by bakanabaka** [comment]
Hey small doubt its asking something like conflicting files , how do I do that

**Comment by yashkite** [comment]
@bakanabaka in your forked repository, just click on the sync fork, and update the branch
image

**Comment by bakanabaka** [comment]
Thanks for the advice man

**Comment by bakanabaka** [comment]
Hey its still showing conflicting files

**Comment by yashkite** [comment]
@bakanabaka because the issue related to the calculator code is already resolved. I have merged the PR and you have some code changes in the calculator which is different from the current code. that's why you are getting an error related to the conflicts. reference #25, to resolve this you can manually add this code to the current code in the repository. By doing this, I think the error will be resolved.

**Comment by yashkite** [comment]
@bakanabaka  don't worry I have resolved the issue by doing the same thing I have mentioned above.

File: Small Projects/Python/Black Jack/run.py [hunk 2]
```diff
@@ -1,186 +1,184 @@
...
         continue
-    money -= pot
-    pot = pot * 2
-    print(f"Your wallet fund is ${money}")
-    print(f"Table: ${pot}")
 
+    money -= pot
+    pot *= 2
 
+    print_stats(computer, user)
```
Comment (yashkite) [inline Small Projects/Python/Black Jack/run.py:66]: image

**Review by yashkite — CHANGES_REQUESTED** [review]
Please resolve the errors

**Review by yashkite — CHANGES_REQUESTED** [review]
image

whenever a player's game is over or a player wins, you have to show which player has which cards and show the sum of the cards also, and if only the player says he like to play another game then only clear the screen or every time player chooses a new card,

**Review by yashkite — CHANGES_REQUESTED** [review]
image

the computer is showing the sum and cards directly to the user, if you don't know how this game work then you can refer [this](https://www.arkadium.com/games/blackjack/) link and play the game.

**Pull request closed without merging**